opentracing-spring-extension/opentracing-spring-webflux-extension/pom.xml

```diff
@@ -40,6 +40,10 @@
                     <groupId>org.glassfish</groupId>
                     <artifactId>jakarta.el</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Why are we excluding this?

Reply:
this is overwriting the spring boot dependencies, because maven takes the first dependency it finds - unless explicitly defined

Reply:
Meaning that this `spring-boot-starter-webflux` brings a version of `spring-boot-autoconfigure` that conflicts with another one. Maven picks this one (or the one from `opentracing-spring-web-starter`, as we're excluding both).
Correct?

Reply:
Yes, `spring-boot-starter-webflux` brings spring boot in `2.3.3.RELEASE`, which overwrites the explicitly declared `<spring-boot.version>2.3.4.RELEASE</spring-boot.version>` everywhere.